### DISCUSSION TITLE
添加百度网盘，在线浏览网盘里的PDF文件、WORD、EXCLE文件，均无法浏览，图片正常，还有视频也不能在线播放。添加的阿里云盘一切正常。

### DISCUSSION THREAD

**Discussion by zhujun99170** [AlistGo/alist#3310] [Q&A]
浏览WORD文件，提示：An error occurred
We're sorry, but for some reason we can't open this for you.；
浏览PDF文件，提示：载入 PDF 时发生错误。PDF.js v3.0.44 (build: 6852001b0)
信息：Failed to fetch；
播放视频文件，连接5次后提示加载失败。

**Comment by imopro** [comment]
视频大部分都是因为浏览器不支持视频编码格式导致的。用视频下面那些播放器播放。
